bounties/BA122401.md

```diff
@@ -0,0 +1,41 @@
+---
+code: BA122401
+title: Rocket Pool Support Payments
```

Comment:
Kind of feels like this straddles grant and bounty. The split total makes sense from GMC POV, but is wildly risky for a bounty hunter... Which kind of makes sense because your audience isn't really bounty hunters, its the existing people doing support work. Not sure this should be on the portal, tbh, but also not sure what else to do with it.

Reply:
As much as I tried to include this on the bounty board, I am ultimately deciding against it. The GMC prefers not to actively solicit for this work, particularly from those outside RP. Their primary motivation for funding was to acknowledge past hard work, not to increase support contributions, especially from external individuals over our existing support contributors.

In terms of intent, this bounty is similar to the Summit Program grant I submitted. Although it is open to multiple participants, it is designed for active community members rather than outsiders.